File: Feb_6th,7th_Tasks/Java programs/ApiResponseTime.java
public class ApiResponseTime {
    public static void main(String[] args) {
        int[] responseTimes = {1200, 2500, 1800, 3000, 900};

        int sum = 0;
        System.out.println("Slow APIs (>2000 ms):");

        for (int time : responseTimes) {
            sum += time;
            if (time > 2000) {
                System.out.println(time);
            }
        }

        double average = (double) sum / responseTimes.length;
        System.out.println("Average Response Time: " + average);
    }
}

File: Feb_6th,7th_Tasks/Python programs/ApiResponseTime.py
response_times = [1200, 2500, 1800, 3000, 900]

slow_apis = [t for t in response_times if t > 2000]
average = sum(response_times) / len(response_times)

print("Slow APIs (>2000 ms):", slow_apis)
print("Average Response Time:", average)
